deps.edn

```diff
@@ -7,7 +7,8 @@
 
  :aliases {:native {:jvm-opts   ["-Dclojure.compiler.direct-linking=true"]
                     :extra-deps {org.clojure/clojure {:mvn/version "1.10.3"}
-                                 org.graalvm.nativeimage/svm {:mvn/version "22.0.0.2"}}}
+                                 ;; NOTE: This needs to match the GRAALVM Version
+                                 org.graalvm.nativeimage/svm {:mvn/version "22.1.0"}}}
```

Comment:
I wonder why we ever added this. I don't think it's necessary.

Reply:
This is required as I add BouncyCastle as a [Feature](https://github.com/babashka/pod-babashka-buddy/pull/4/files#diff-f47055c890e398ca8c9c358ea148662bc5a2d2e3ab86ea33c1f5c43e46c5282c)